app/views/posts/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="card">
+  <div class="card-header">
+    <%= @post.title %>
+  </div>
+  <div class="card-body">
+     <p class="card-text"><%= @post.body %></p>
+    <%= link_to 'Back', posts_path,class: "btn btn-primary" %>
+    <% if user_signed_in? && @post.user.id == current_user.id %>
+    <%= link_to 'Edit', edit_post_path(@post),class: "btn btn-primary" %>
+    <%= link_to 'Destroy', @post, method: :delete, data: { confirm: 'Are you sure?' },class: "btn btn-danger" %>
+    <% end %>
```

Comment:
This is another logic that must be moved to a helper file.

Reply:
gg